tests/lwmac/README.md

```diff
@@ -1,4 +1,4 @@
-examples/default
+LWMAC test application
 ================
```

Comment:
Minor nitpick: can you extend the `=`s to the length of the new title, please?

Reply:
Yap!~